trace/trace.go

```diff
@@ -430,21 +452,6 @@ func (sc SpanContext) WithTraceFlags(flags byte) SpanContext {
 	}
 }
 
-// IsDeferred returns if the deferred bit is set in the trace flags.
-func (sc SpanContext) IsDeferred() bool {
-	return sc.traceFlags&FlagsDeferred == FlagsDeferred
-}
-
-// IsDebug returns if the debug bit is set in the trace flags.
-func (sc SpanContext) IsDebug() bool {
-	return sc.traceFlags&FlagsDebug == FlagsDebug
-}
-
-// IsSampled returns if the sampling bit is set in the trace flags.
-func (sc SpanContext) IsSampled() bool {
-	return sc.traceFlags&FlagsSampled == FlagsSampled
-}
```

Comment:
Should we keep this as a convenience method? It will cut down a bit on the verbosity of determining if a `SpanContext` is sampled.

Also, though not paramount given the pre-1.0 state of the project, it would help those already adopting the codebase.